src/pages/FolderPage/components/FolderSortBar.js

```diff
@@ -0,0 +1,24 @@
+import SortButton from './SortButton';
+import '../styles/folderSortBar.css';
+import FolderAddButton from './FolderAddButton';
+
+function FolderSortBar({ folders, handleClick, selectedId }) {
+  return (
+    <div className="folder__sort-bar">
+      <div className="folder__sort-bar--sort-buttons">
+        {folders.map((folder) => (
+          <SortButton
+            key={folder.id}
+            id={folder.id}
+            text={folder.name}
+            handleClick={handleClick}
+            isclickedProp={selectedId == folder.id ? true : false}
```

Comment:
```suggestion
            isclickedProp={selectedId === folder.id}
```

이렇게만 하셔도 true랑 false 반환이 가능합니다! 
만약 selectedId와  folder.id의 type이 달라 의도적으로 ==을 사용하신거라면, 1순위로는 둘의 타입을 되도록 맞춰주시는게 좋고, 그게 불가능한 경우라면 ==을 사용하시는걸 추천드립니다

Reply:
추가로 isClickedProp과 같이 네이밍 지켜주세요~!!

Reply:
```suggestion
            isClicked={selectedId == folder.id ? true : false}
```

이렇게 하면 사용하는 쪽에서도 네이밍이 통일돼서 좋을 것 같군요